source/parser/parser.cpp

```diff
@@ -39,6 +39,8 @@
 // C++ variants of C standard header files
 #include <cctype>
 #include <cmath>
+#include <cstdarg>
```

Comment:
`<cstdarg>` is explicitly included here (and in other places) as `va_start` and `va_end` are declared there.